src/compass/s1_geocode_slc.py

```diff
@@ -89,43 +118,55 @@ def run(cfg: GeoRunConfig):
             burst.slc_to_vrt_file(temp_slc_path)
             rdr_burst_raster = isce3.io.Raster(temp_slc_path)
 
-        # Generate output geocoded burst raster
-        geo_burst_raster = isce3.io.Raster(
-            f'{burst_output_path}/{id_pol}.slc',
-            geo_grid.width, geo_grid.length,
-            rdr_burst_raster.num_bands, gdal.GDT_CFloat32,
-            cfg.geocoding_params.output_format)
-
         # Extract burst boundaries
         b_bounds = np.s_[burst.first_valid_line:burst.last_valid_line,
                    burst.first_valid_sample:burst.last_valid_sample]
 
         # Create sliced radar grid representing valid region of the burst
         sliced_radar_grid = burst.as_isce3_radargrid()[b_bounds]
 
-        # Geocode
-        isce3.geocode.geocode_slc(geo_burst_raster, rdr_burst_raster,
-                                  dem_raster,
-                                  radar_grid, sliced_radar_grid,
-                                  geo_grid, orbit,
-                                  native_doppler,
-                                  image_grid_doppler, ellipsoid, threshold,
-                                  iters, blocksize, flatten,
-                                  azimuth_carrier=az_carrier_poly2d)
-
-        # Set geo transformation
-        geotransform = [geo_grid.start_x, geo_grid.spacing_x, 0,
-                        geo_grid.start_y, 0, geo_grid.spacing_y]
-        geo_burst_raster.set_geotransform(geotransform)
-        geo_burst_raster.set_epsg(epsg)
-        del geo_burst_raster
-        del dem_raster # modified in geocodeSlc
-
-        # Save burst metadata
-        metadata = GeoCslcMetadata.from_georunconfig(cfg, burst_id)
-        json_path = f'{burst_output_path}/{id_pol}.json'
-        with open(json_path, 'w') as f_json:
-            metadata.to_file(f_json, 'json')
+        # Create top output path
+        burst_output_path = f'{cfg.product_path}/{burst_id}/{date_str}'
+        os.makedirs(burst_output_path, exist_ok=True)
+
+        '''
```

Comment:
Not sure why this part is commented out, shouldn't it be part of the writing to HDF5?

Reply:
Was commented out to test something and forgot to uncomment.